make.ps1

```diff
@@ -186,13 +143,16 @@ elseif ($command -eq "check")
 		./OpenRA.Utility.exe cnc --check-code-style OpenRA.Utility
 		echo "Checking for code style violations in OpenRA.Test..."
 		./OpenRA.Utility.exe cnc --check-code-style OpenRA.Test
+		echo "Checking for explicit interface violations..."
+		./OpenRA.Utility.exe all --check-explicit-interfaces
```

Comment:
not sure, i see that this is the exact opposite of the last commit before me, so either i used an old version by mistake, or alternatively i remember seeing some errors when i was testing all the commands so i may have fixed something and forgot about it. 

will return back for now